sdcm/coredump.py

```diff
@@ -202,11 +202,11 @@ def extract_info_from_core_pids(
     def _upload_coredump(self, core_info: CoreDumpInfo):
         coredump = core_info.corefile
         coredump = self._pack_coredump(coredump)
-        base_upload_url = 'upload.scylladb.com/%s/%s'
+        base_upload_url = 'https://upload.scylladb.com/%s/%s'
         coredump_id = os.path.basename(coredump)[:-3]
         upload_url = base_upload_url % (coredump_id, os.path.basename(coredump))
         self.log.info('Uploading coredump %s to %s' % (coredump, upload_url))
-        self.node.remoter.run("sudo curl --request PUT --upload-file "
+        self.node.remoter.run("sudo curl --request PUT --fail --show-error --upload-file "
```

Comment:
`--fail` will cause additional error (critical?).

Maybe we should add `ignore_status=True` and later log error message if failed?